modules/ROOT/attachments/cloud-api.yaml

```diff
@@ -5195,7 +5195,7 @@ paths:
         - CloudStorageService
   /v1alpha2/cloud-storage/topics/unmount:
     post:
-      description: Unmount topics to object storage, freeing up all local cluster resources. Once you unmount a topic, it can no longer be consumed or produced to. It detaches from the active cluster while its data remains safely stored in the external object storage.
+      description: Unmount topics to object storage, freeing up all local cluster resources. Once you unmount a topic, it can no longer be consumed or produced to. It detaches from the active cluster while its data remains safely stored in the external Object Storage.
```

Comment:
```suggestion
      description: Unmount topics to object storage, freeing up all local cluster resources. Once you unmount a topic, it can no longer be consumed or produced to. It detaches from the active cluster while its data remains safely stored in the external object storage.
```